src/test/groovy/com/github/vkorobkov/jfixtures/config/structure/tables/TablesTest.groovy

```diff
@@ -0,0 +1,71 @@
+package com.github.vkorobkov.jfixtures.config.structure.tables
+
+import com.github.vkorobkov.jfixtures.config.yaml.Node
+import spock.lang.Specification
+
+class TablesTest extends Specification {
+    def SAMPLE_CONFIG = [
+            table_NOT_have_pk: [
+                    applies_to: "friends",
+                    pk: [generate: false],
+            ],
+            users_related_table_has_pk: [
+                    applies_to: "users",
+                    pk: [generate: true],
+        ]
+    ]
+
+    def SAMPLE_CONFIG_REGEXP = [
+            tables_NOT_have_pk: [
+                    applies_to: "/.+",
+                    pk: [generate: false],
+            ],
+            table_has_pk: [
+                    applies_to: "users",
+                    pk: [generate: true],
+            ]
+    ]
+
+    def SAMPLE_CONFIG_WITH_CUSTOM_PK = [
+            table_has_pk: [
+                    applies_to: "users",
+                    pk: [generate: true, column: "custom_id"],
+            ]
+    ]
+
+    def "should auto generate PK when generate flag does not exist"() {
```

Comment:
we probably need to rename the method like `shouldAutoGeneratePk returns true by default`